Eatery Blue/Eatery Blue/UI/AlertMessage/AlertMessageView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AlertMessageView.swift
+//  Eatery Blue
+//
+//  Created by William Ma on 1/28/22.
+//
+
+import UIKit
+
+class AlertMessageView: UIView {
+
+    private let stackView = UIStackView()
+    let imageView = UIImageView()
+    let messageLabel = UILabel()
+
```

Comment:
I think we should follow convention you used for most of codebase. Did you alphabetize variables you instantiate as fields?

Reply:
It seems like you didn't, so that's fine. Is there anything to keep in mind when creating fields in the codebase?

Reply:
I tried to order/group the fields in a way that made sense, otherwise alphabetically. For views and view controllers this meant top-to-bottom or left-to-right in the view.

Reply:
Got it, makes sense.